### ISSUE TITLE
Trading bank and on landing screens not displaying correctly

### ISSUE THREAD

**Issue by IsocratesGrey** [endless-sky/endless-sky#9188]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When arrival at a planet the welcome screen as well as the trading and bank screen are not properly centered. I tried removing all preference and doing a clean load. I'm using the newest stable version from github. Tried both on just the laptop as well as on a multi-monitor setup.

### Steps to Reproduce

1. Go to any planet
2. If the text is not taking up the full text box you have the same problem as me.

### Expected Behavior

Text taking up the textbox correctly and centred correctly.

### Screenshots

![es1](https://github.com/endless-sky/endless-sky/assets/119460883/6ceb6948-70be-49c8-a119-4fcbb013f044)
![es2](https://github.com/endless-sky/endless-sky/assets/119460883/b6b01dad-f21b-4132-a653-c99506992395)

### Operating System

Windows 10 Enterprise 10.0.19042 Build 19042

### Game Source

GitHub Releases

### Game Version

0.10.2

**Comment by TomGoodIdea** [comment]
Managed to reproduce (on Windows, using the default shell to extract the game zip, with 0.9.16.1 updated to 0.10.2): if you update your game just by extracting the game into where the previous version was, without removing old files, you can end up with a state where files named same in the old and the new version are overwritten, but files renamed (or moved), like `interfaces.txt` moved to `_ui` subfolder, are present from both versions.
The game loads `data/_ui/interfaces.txt` first and then `data/interfaces.txt`. The outdated interfaces overwrite the newer ones, which here results in a broken UI.

TLDR: try removing the whole game installation folder and then reinstalling.

**Comment by Coolwaterdragon** [comment]
just a note for anyone on Steam experiencing the same issue, only uninstalling through Steam won't work, you have to find the folder like Tom was saying, and manually delete a few things that get left in there.

**Comment by quyykk** [comment]
@Coolwaterdragon did you experience this issue using Steam?

**Comment by Coolwaterdragon** [comment]
Yes it was on steam

**Comment by ziproot** [comment]
See https://github.com/endless-sky/endless-sky/pull/9491#issuecomment-1773912576
> A majority of the reports for this issue have been from people who used the ES launcher to update to continuous, which doesn't delete old, no longer existent files. The reproduction method that Tom mentioned in #9188 similarly didn't delete old files, therefore causing an incorrect interfaces file to be used.
> 
> There's only one report of this occurring on Steam, but Steam actually does delete old files when updating, and if it didn't then we would have received far more reports of this being an issue when 0.10.1 and 0.10.2 were released.
> 
> So insofar as it's an installation failure (be that the method used doesn't delete old files or Steam seemingly bugged out), I don't see there being anything to actually fix on our end.

**Comment by Amazinite** [comment]
The fix is to do a clean installation of the latest version, or to delete the old versions of the files that were moved by #8044.

**Comment by TomGoodIdea** [comment]
If we distribute the Windows version of the game zipped, we should take into account that people use different software to unpack it and that the software may not support deleting the old files.

Recommending a clean install on game upgrade in readme or the wiki won't hurt, probably. Here the problem was noticed, because it broke the planet UI, but it could affect every other moved data file.

If we want to make sure the game gets upgraded correctly, an obvious solution is to have an installer for Windows.
